sos/options.py

```diff
@@ -282,6 +282,9 @@ def filter_opt(name, value):
             """
             if name in ("add_preset", "del_preset", "desc", "note"):
                 return False
+            # Exception list for options that still need to be reported when 0
+            if name in ['log_size'] and value == 0:
```

Comment:
Could we add here also `plugin_timeout`? It suffers by the same issue.

Otherwise great catch in both places. All my testing passed.

Reply:
Ack